packages/l10n/src/create-l10n-injector/create-l10n-injector.tsx

```diff
@@ -2,19 +2,21 @@ import React from 'react';
 import { reportErrorToSentry } from '@commercetools-frontend/sentry';
 import { getDisplayName } from '../utils';
 
-type LoadLocale<LoadedData> = (locale: string) => Promise<LoadedData>;
-type InjectorOptions<LoadedData> = {
+type LoadLocale<LoadedData extends {}> = (
```

Comment:
The `extends {}` is unrelated to the issue, but is still useful to indicate that the generic type should be an object